### PR TITLE
Work and about section

### PR DESCRIPTION
**Pull request by usorfaitheloho**
## Milestone 2 

in this milestone, I worked on the following :
- Created work section
- created about myself section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,6 +52,250 @@ <h1>Hey there. I’m Mariane<br />
...
+                            <li class="tech-list">
+                                <p class="tech-name">css</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">JavaScript</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">html</p>
+                            </li>
+                        </ol>
```
Comment (rustam-tolipov) [inline index.html]: ## BEAUTIFUL DESIGN 😊👍
- [ ] Good job so far, unfortunately, some issues need to be done 🥲. Please change all your `<ol>` tags to `<ul>`. 

These tags should be fixed 👍
Screen Shot 2022-01-15 at 3 04 19 PM

File: index.html [hunk 2]
```diff
@@ -50,6 +52,250 @@ <h1>Hey there. I’m Mariane<br />
...
+                    </div>
+                </div>
+            </div>
+            <div class="about-me">
+                <h1>About me</h1>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <button class="btn btn-projects"  type="button">Get my resume</button>
+            </div>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] In your about-me `div` you forgot to add a background image. Let's fix it by adding a proper background image using CSS. TIP 📌: Use `background-image: ...;` property in CSS as you have done in your hero section 🤓🍀
#### Your design
Screen Shot 2022-01-15 at 3 10 28 PM

#### Figma design
Screen Shot 2022-01-15 at 3 16 26 PM

File: index.html [hunk 3]
```diff
@@ -50,6 +52,250 @@ <h1>Hey there. I’m Mariane<br />
...
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Gitlab</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Terminal</p>
+                        </li>
+                    </ol>
+                </div>
+            </div>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] And lastly, please wrap your about me part into one section element. In your code from the line `189` to `273` 🍀😀. If you do all the suggestions I gave you, you should be approved by the next reviewer 😊

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @usorfaitheloho , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Whoistolu) [reply index.html]: Well implemented 👍🏽

File: index.html (hunk 2 above)
Comment (Whoistolu) [reply index.html]: Well implemented 👍🏽

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html]: Well implemented 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @usorfaitheloho ,

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 1 above)
Comment (usorfaitheloho) [reply index.html]: Thanks @Rustamxon7 ,I have changed all affected tags as suggested

File: index.html (hunk 2 above)
Comment (usorfaitheloho) [reply index.html]: Hello @Rustamxon7 ,
Thanks for your review 😄 . I have added the background image in the about-me section

File: index.html (hunk 3 above)
Comment (usorfaitheloho) [reply index.html]: Hi @Rustamxon7 

I have wrapped the about-me into a section.
Thanks for your review 👍

**Comment by usorfaitheloho** [comment]
Thanks for your review 😄  @Whoistolu

**Pull request merged**